src/client/components/CreateSession/CreateSessionTextFields.jsx

```diff
@@ -0,0 +1,169 @@
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import { Tooltip } from "@mui/material";
+import { useDispatch } from 'react-redux';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { createNewSession } from "../../actions";
+import './CreateSession.css'
+import React from "react";
+
+const equipmentInfo = (
+  <React.Fragment>
+    <div>Enter information about the equipment in the following format:</div>
+    <div style={{textAlign: "center"}}>"Quantity, Equipment Name"</div><br />
+    <div>Example:</div>
+    <div>1, Baseball Bat</div>
+    <div>4, Tennis Ball</div>
+    <div>2, Hockey Stick</div>
+    <div>1, Volleyball Net</div>
+  </React.Fragment>
+)
+
+export default function CreateSessionTextFields() {
+  const [session_name, setName] = useState("");
+  const [session_sport, setSport] = useState("");
+  const [session_description, setDescription] = useState("");
+  const [session_city, setCity] = useState("");
+  const [session_location, setLocation] = useState("");
+  const [session_equipment_needed, setEquipmentNeeded] = useState("");
+  const [session_players_needed, setPlayersNeeded] = useState("");
```

Comment:
out of curiosity, can't this be handled in just one state?

Reply:
All the tutorials I found online use multiple `useState` function calls, so I'm not sure. If you know of a way to do it with one function call, please let me know and I'll refactor it.

Reply:
const [session, setSession] = useState({
  session_name: "",
  session_sport: "",
  session_description: "",
  session_city: "",
  session_location: "",
  session_equipment_needed: "",
  session_players_needed: ""
});

I think to set the state you have to use the spread operator
setSession(prevState => ({
  ...prevState,
  session_name: "new session"
}));

tbh either way is fine but this way might reduce the code and already have all items in one object

Reply:
Thank you React sensei 🙇🏼 

Changed in the last commit